extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzRuntimeConfig.java

```diff
@@ -19,4 +19,12 @@
     @ConfigItem(defaultValue = "5")
     public int threadPriority;
 
+    /**
+     * By default, the scheduler is not started unless a {@link io.quarkus.scheduler.Scheduled} business method is found.
+     * If set to true the scheduler will be started even if no scheduled business methods are found. This is necessary for
+     * "pure" programmatic scheduling.
+     */
+    @ConfigItem(defaultValue = "false")
```

Comment:
`defaultValue` is not necessary. `boolean` config items defaults to `false`

Reply:
I prefer explicit values so I'd like to keep it as it is.